acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/utils/Eth2BlockSigningRequestUtil.java

```diff
@@ -32,7 +32,7 @@
 
 public class Eth2BlockSigningRequestUtil {
   private final SpecMilestone specMilestone;
-  private final DataStructureUtil dataStructureUtil;
+  private final BeaconBlockUtil beaconBlockUtil;
```

Comment:
Is DataStructureUtil still needed?

Reply:
No, its wrapped in BeaconBlockUtil class.